cmake/DAGMCConfig.cmake.in

```diff
@@ -0,0 +1,22 @@
+# - Config file for the Direct Accelerated Geometry Monte Carlo (DAGMC) package
+# It defines the following variables
+#  DAGMC_INCLUDE_DIRS - include directories for FooBar
+#  DAGMC_LIBRARIES    - libraries to link against
```

Comment:
Perhaps we don't need these comments unless we plan to always update them to cover all variables defined in this file

Reply:
I think they're kinda useful, but only if the descriptions are improved.